### PR TITLE
Added bullet icon

### PR DESCRIPTION
**Pull request by mdulat**

### CODE REVIEW

**Comment by mdulat** [comment]
Not sure why CI failed, something about fill property, any help would be appreciated.

File: images/tier1/bullet.svg [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (dlockhart) [inline images/tier1/bullet.svg]: I'd run this through https://jakearchibald.github.io/svgomg/ to clean it up and make sure you follow all the instructions in the README about sizes and such.

**Comment by mdulat** [comment]
Ok, svg file fixed, also added 2 more icons. Please let me know if these are ok.

File: images/tier1/arrow-expand-small.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<svg width="18" height="18" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 18 18" mirror-in-rtl="true">
+  <path fill="#72777a" d="M7.708,2.29 C7.518,2.1 7.258,2 6.998,2 L-0.998,2 C-1.258,2 -1.518,2.1 -1.708,2.29 C-2.098,2.69 -2.098,3.32 -1.708,3.71 L2.29,7.71 C2.49,7.9 2.74,8 3,8 C3.26,8 3.51,7.9 3.71,7.71 L7.708,3.71 C8.097,3.32 8.097,2.69 7.708,2.29 L7.708,2.29 Z"/>
+</svg>
```
Comment (dlockhart) [inline images/tier1/arrow-expand-small.svg]: I don't think this one is quite correct... when I view it, it appears to just be the collapse icon but cut off a bit. I think it should be an arrow opening to the right?

**Comment by dlockhart** [comment]
The SCSS files and the [markdown](https://github.com/BrightspaceUI/icons/blob/master/d2l-icons.md) file should have been auto-generated and updated, did you run the build script?

**Comment by mdulat** [comment]
Dave, can you please merge this if it's ok, as I don't have write access to merge. Thanks!

**Comment by dlockhart** [comment]
@mdulat Done, and I've released a new version as well -- `2.17.0`.

**Pull request merged**